cypress/integration/seededFlows/podcastFlows/followPodcast.spec.js

```diff
@@ -10,26 +10,22 @@ describe('Follow podcast', () => {
   it('Follows and unfollows a podcast', () => {
     cy.get('[data-follow-clicks-initialized]');
 
-    cy.findByRole('heading', {
-      name: 'Developer on Fire Developer on Fire Follow',
-    });
```

Comment:
This is here to check that we're on the right page before checking the functionality - I don't think it's a big deal, but it could be useful to leave it in.